gcs_oauth2_boto_plugin/oauth2_client.py

```diff
@@ -258,14 +260,17 @@ def GetToken(self, key):
     except (IOError, OSError) as e:
       if e.errno != errno.ENOENT:
         LOG.warning('FileSystemTokenCache.GetToken: '
-                    'Failed to read cache file %s: %s', cache_file, e)
+                    'Failed to read cache file %s: %s',
+                    six.ensure_text(cache_file),
+                    six.ensure_text(e))
```

Comment:
Shouldn't you call `str()` on the thing before passing it to `ensure_text`?  `e` is an exception, and `ensure_text` throws a `TypeError` if the arg you pass isn't bytes or unicode.